sample-android-app/src/main/java/com/badoo/reaktive/sample/android/MainActivity.kt

```diff
@@ -24,11 +25,11 @@ class MainActivity : AppCompatActivity() {
 
         findViewById<Button>(R.id.button).setOnClickListener {
             observable<String> { emitter ->
```

Comment:
Also it can be replaced with 'observableFromFunction'. So just sleep and return the value.

Reply:
Done